backend/handler/getDBUri.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Do we really have to move this out?

The handler folder is suppose to hold handlers to external services (e.g. mongodb, S3 etc), it's a bit weird to have a file called `getDBUri`, which should technically reside in `db.js`